contrib/python-certifi/template.py

```diff
@@ -0,0 +1,19 @@
+pkgname = "python-certifi"
```

Comment:
this needs a patch to pick up the system ca bundle instead of using the one that comes with it, see e.g. alpine etc